backend/app.js

```diff
@@ -61,36 +61,38 @@ app.use(bodyParser.json());
 // Use CORS middleware
 app.use(
 	cors({
-		origin: "http://localhost:8080", // Replace with your frontend's URL
+		//origin: "http://localhost:8080", // Replace with your frontend's URL
+		origin: "pblank.blewi.xyz", // Replace with your frontend's URL
```

Comment:
You shouldn't need to hard-code this in here, just change the env file to use that config.